rdmo/projects/tests/test_handlers_tasks.py

```diff
@@ -0,0 +1,151 @@
+
+from django.contrib.auth.models import Group
+
+from rdmo.projects.models import Project
+from rdmo.tasks.models import Task
+
+from .helpers import assert_other_projects_unchanged
```

Comment:
I like this. We should stick to `helpers` for tests (and use sparingly) and `utils` for regular code.